add checkbox solo example (#2971)
grommet/hpe-design-system: aries-site/src/pages/components/checkbox.mdx

### CheckBox outside of FormField

A checkbox might be used outside of a form field when:
1. Serving as a UI control showing or hiding page content, such as revealing additional content or inputs which are only relevant when a particular feature is enabled.

> 2. Scenario's in which having a formfield label in addition to the checkbox label would is to redundant for the user. 

Reviewer: ```suggestion
2. Scenarios in which having a form field label in addition to the checkbox label would be redundant for the user. 
```

### CheckBox outside of FormField

A checkbox might be used outside of a form field when:
1. Serving as a UI control showing or hiding page content, such as revealing additional content or inputs which are only relevant when a particular feature is enabled.
2. Scenario's in which having a formfield label in addition to the checkbox label would is to redundant for the user. 

> 2. There is no additional information or help text necessary for the user to understand what is being asked of them.

Reviewer 2: I think this can go now that we have the new #2.

```suggestion
```

Author: opps yes forgot that commit

### CheckBox outside of FormField

In product design there is usage for a single checkbox that may be needed to turn a feature on or off.

> At times a single Checkbox may control progressiive disclosure which works well if the checkbox is not wrapped in a FormField.

Author: yes working on rewording thank you for these ideas it helps

### CheckBox outside of FormField

> 

Reviewer 3: Thinking that we should include a brief description of what the visual changes are for a checkbox outside of a formfield here...

"CheckBox can also be used outside of a FormField. When doing so, the CheckBox does not have a border or label. Here are some common use cases for when a CheckBox can be used outside of a FormField: "

Reviewer 4: Agreed ^^

Author: done

> 1. When there is progressive disclosure: i.e. clicking on the checkbox will reveal related options that wouldn't have been displayed unless the user checked the box.

Reviewer 3: Wondering if its worth it to show a visual for this use case?

Author: yeah thinking maybe on the Forms page? maybe we can add to an existing example?

Author: I'll make a ticket for us to work on this separately on the forms page like we discussed.

Author: https://github.com/grommet/hpe-design-system/issues/2981

> 2. To turn a feature on or off also consider using a [toggle button](https://design-system.hpe.design/components/button#toggle-buttons)

Reviewer 3: Not sure if this bullet is correctly placed. Seems out of line with the other items which are use cases specifically for Checkboxes outside of a FormField.

### CheckBox outside of FormField

> When using CheckBox outside of Formfield, it does not have a border or label. The hover indentifier will also be on the CheckBox itself.

Author: good point yes just a detail but agree should be with pad below

> Here are some common use cases for when a CheckBox can be used outside of a FormField:

Reviewer 2: Only capitalize when we are referencing the actual component (proper noun). Sentence case when writing about the concept.

```suggestion
A checkbox might be used outside of a form field when:
```

> 1. When there is progressive disclosure: i.e. clicking on the checkbox will reveal related options that wouldn't have been displayed unless the user checked the box.

Reviewer 2: i.e. means "in other words"
e.g. means "for example"

I believe we want "for example" here because we are providing an example of progressive disclosure.

Personally, I think "progressive disclosure" is still too narrow and would prefer [my suggestion](https://github.com/grommet/hpe-design-system/pull/2971#discussion_r997479711) speaking to the checkbox is serving as a UI control. Medium-soft opinion.

```suggestion
1. There is progressive disclosure (e.g. clicking on the checkbox will reveal related options that wouldn't have been displayed unless the user checked the box).
```

Author: that's fair Kenny and I did also chat about being more detailed

Author: done

> 2. There is no additional information or help text necessary for the user to understand what is being asked of them.

Reviewer 2: I am having a hard time picturing this. Can someone provide me an example for where this might occur?

Author: are you referring to number 2?

3. Allowing a user to opt-in or acknowledge a disclaimer, such as a privacy policy or terms and conditions.

> 

Reviewer 2: Can I use a checkbox outside a form field for selecting an item from a list?

Author: umm not sure I can picture what you are referring to?

Author: - Allowing an item to be selected within a list, table row, or drop down.
☝️ do we use `checkbox` in a list? In a DataTable checkbox is built in so just trying to understand where this case would be used cant visualize it
